sites/ava-site/examples/lite-insight/custom/demo/homogeneous.jsx

```diff
@@ -41,7 +41,9 @@ const App = () => {
       <Spin spinning={loading} style={{ marginTop: 80 }}>
         <div style={{ width: '100%' }}>
           {result.homogeneousInsights &&
```

Comment:
How about changing it to `result.homogeneousInsights?.map((item, index) => (<InsightCard key={index} insightInfo={item} height={400} />))`